compiler/qsc_frontend/src/compile/tests.rs

```diff
@@ -1267,7 +1267,7 @@ fn accept_use_qubit_block_syntax_if_preview_feature_is_off() {
         &store,
         &[(std, None)],
         sources,
-        TargetCapabilityFlags::empty(),
+        TargetCapabilityFlags::all(),
```

Comment:
This change doesn't seem like it would be needed... does something fail without it?